lib/tasks/big_query.rake

```diff
@@ -19,7 +22,19 @@ namespace :big_query do
     classes.each do |c|
       puts "Queueing: #{c.count} #{c} entities"
 
-      c.find_each(batch_size: 200, &:send_import_event)
+      c.find_each(batch_size: 200) do |entity|
+        attempt_count = 0
+        begin
+          entity.send_import_event
+        rescue Redis::CommandError
+          attempt_count += 1
+          abort if attempt_count > ON_REDIS_ERROR_RETRY_ATTEMPTS
+
+          puts "sleeping for #{ON_REDIS_ERROR_WAIT_TIME_IN_SECONDS} - redis error"
```

Comment:
If I'm not mistaken, this kind of thing will get picked up by the GPaaS log processor and sent to Logit, which expects a JSON string. If we're going to emit this message it may be better to make it a proper log message.